### PR TITLE
Ziafazal/yonk 695

### PR DESCRIPTION
**Pull request by aamishbaloch**
Configuration Pull Request
---

Make sure that the following steps are done before merging:

  - [ ] A DevOps team member has approved the PR.
  - [ ] Are you adding any new default values that need to be overridden when this change goes live? If so:
    - [ ] Update the appropriate internal repo (be sure to update for all our environments)
    - [ ] If you are updating a secure value rather than an internal one, file a DEVOPS ticket with details.
    - [ ] Add an entry to the CHANGELOG.
  - [ ] Have you performed the proper testing specified on the [Ops Ansible Testing Checklist](https://openedx.atlassian.net/wiki/display/EdxOps/Ops+Ansible+Testing+Checklist)?

### CODE REVIEW

**Comment by aamishbaloch** [comment]
I have completed the checklist. Link for my sandbox is aamishbaloch.sandbox.edx.org.

**Comment by aamishbaloch** [comment]
@feanil Can you please review this?

File: playbooks/roles/edxapp/defaults/main.yml [hunk 1]
```diff
@@ -1311,3 +1318,6 @@ SERVICE_WORKER_USERS:
     username: "{{ EDXAPP_ENTERPRISE_SERVICE_WORKER_USERNAME }}"
     is_staff: true
     is_superuser: false
+
+# The age at which a learner no longer requires parental consent, or None
+EDXAPP_PARENTAL_CONSENT_AGE_LIMIT: 13
```
Comment (jibsheet) [inline playbooks/roles/edxapp/defaults/main.yml]: Where is this variable used?

Comment (aamishbaloch) [reply playbooks/roles/edxapp/defaults/main.yml]: common/djangoapps/student/views.py:2275
common/djangoapps/student/models.py:368

@jibsheet here are the uses in edx-platform.

Comment (jibsheet) [reply playbooks/roles/edxapp/defaults/main.yml]: I should have been more specific.
Where in this pull request do you use this variable.

You add EDXAPP_PROFILE_IMAGE_SIZES_MAP to lms_env_config, you don't seem to do the same thing with this var (it's just going to be dropped on the floor).

Comment (aamishbaloch) [reply playbooks/roles/edxapp/defaults/main.yml]: @jibsheet I have added it to lms_env_config.

Comment (feanil) [reply playbooks/roles/edxapp/defaults/main.yml]: thanks @aamishbaloch , all caps vars should be above any lower case vars.  Can you please move this declaration higher in the file?

Comment (aamishbaloch) [reply playbooks/roles/edxapp/defaults/main.yml]: @feanil Done

**Review by jibsheet — APPROVED** [review]

**Pull request merged**